src/StickyParallaxHeader.js

```diff
@@ -219,13 +219,14 @@ class StickyParallaxHeader extends Component {
   }
 
   renderForeground = (backgroundHeight) => {
-    const { foreground, tabsContainerBackgroundColor } = this.props
+    const { foreground, tabsContainerBackgroundColor, backgroundImage } = this.props
 
     return (
       <View
         style={{
           height: backgroundHeight,
-          backgroundColor: tabsContainerBackgroundColor
+          backgroundColor: tabsContainerBackgroundColor,
+          ...( backgroundImage && styles.transparentBackground )
```

Comment:
Does this logic assume that image must always have a background itself. What if someone adds a logo as a background? Should logic not be that by default for image: background color is transparent, however, a useer can override it by passing custom backgroundColor? Just out of curiousity ;-)

Reply:
Currently backgroundImage is under foreground component, that’s why It’s positioned absolutely and with transparent background color. To set custom background color under backgroundImage we need another prop - adding this idea to backlog.